compiler/luci/service/src/Nodes/CircleRelu.test.cpp

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
@llFreetimell I think this change is a little bit weird. For now, the file `service/src/Nodes/${NODE_NAME}.cpp` implements "service" function that may be use in passes or sth. And `test.cpp` file tests its `service` function. So, this contents should be moved to `CircleTypeInference.test.cpp` or sth because it tests `luci::tinf::Rule` and so should `luci::sinf::Rule`.

Reply:
I am planning to restructure `luci/Service` as following structure.
```
- service/src
  - Nodes
    - CircleRelu.cpp : implements shape/type inference rule for Relu
    - CircleRelu.test.cpp : includes shape/type inference rule tests for Relu
    - ...
  - CircleShapeInference.cpp
  - CircleTypeInference.cpp
  - ...
```
And `CircleShapeInference.cpp` will include only `infer` function like https://github.com/Samsung/ONE/blob/f026107f9c182acb26d25ba0f6dbf4fadce41119/compiler/luci/service/src/CircleShapeInference.cpp (So does `CircleTypeInference.cpp`).

As a result, for me, implementing those tests in `CircleTypeInference.test.cpp` or `CircleShapeInference.test.cpp` looks weird because rules are collected at one file but tests are separated :(